host-spec/c03-transition.tm

```diff
@@ -258,34 +244,31 @@
 
   <subsection|Transactions>
 
-  <subsubsection|Transaction Submission>
-
-  Transaction submission is made by sending a <em|Transactions> network
-  message. The structure of this message is specified in Section
-  <reference|sect-msg-transactions>. Upon receiving a Transactions message,
-  the Polkadot Host decodes the SCALE-encoded blob and decouples the
-  transactions into individually SCALE-encoded transactions. Afterward, it
-  should call <verbatim|validate_trasaction> Runtime entry on each individual
-  transaction, defined in Section <reference|sect-rte-validate-transaction>,
-  to check the validity of the received transaction. If
-  <verbatim|validate_transaction> considers the submitted transaction as a
-  valid one, the Polkadot Host makes the transaction available for the
-  consensus engine for inclusion in future blocks.
-
-  <subsection|Transaction Queue>
-
-  A Block producer node should listen to all transaction
-  messages<em|<index|Transaction Message>>. The transactions are submitted to
-  the node through the <em|transactions> network message specified in Section
-  <reference|sect-msg-transactions>. Upon receiving a transactions message,
-  the Polkadot Host separates the submitted transactions in the transactions
-  message into individual transactions and passes them to the Runtime by
-  executing Algorithm <reference|algo-validate-transactions> to validate and
-  store them for inclusion into future blocks. Valid transactions are
-  propagated to connected peers of the Polkadot Host. Additionally, the
+  Transaction are submitted and exchanged through <em|Transactions> network
+  messages (see Section <reference|sect-msg-transactions>). Upon receiving a
+  Transactions message, the Polkadot Host decodes the SCALE-encoded blob and
+  splits it into individually SCALE-encoded transactions.
+
+  Alternative transaction can be submitted to the host by offchain worker
+  through the <verbatim|ext_offchain_submit_transaction> Host API, defined in
+  Section <reference|sect-ext-offchain-submit-transaction>.\ 
+
+  Any new transaction should be submitted to the
+  <verbatim|validate_transaction> Runtime function, defined in Section
+  <reference|sect-rte-validate-transaction>. This will allow the Polkadot
+  Host to check the validity of the received transaction against the current
+  state as well as determine how the transaction depends on other extrinsics
```

Comment:
> the transaction depends on other extrinsics

again, I don't understand this sentence :thinking:

Reply:
`validate_transaction` returns this information, so I thought it should be mentioned. I will remove it for now.